base_custom_info/README.rst

```diff
@@ -26,7 +37,7 @@ expected for a given record.
 
 To define a template, you need to:
 
-* Go to *Settings > Custom Info > Templates*.
+* Go to *Custom Info > Templates*.
 * Create one.
 * Add some *Properties* to it.
```

Comment:
I would like a definition of "property" in this context. This is important as "property" has a special meaning in Odoo, and I'm not sure these are the same (i.e. fields with a company dependent value)